packages/@ember/-internals/runtime/lib/mixins/array.js

```diff
@@ -658,6 +658,15 @@ const ArrayMixin = Mixin.create(Enumerable, {
     object that will be set as `this` on the context. This is a good way
     to give your iterator function access to the current object.
 
+    Example Usage:
+
+    ```javascript
+      let foods = ['apple', 'banana', carrot'];
+
+      foods.forEach(food => console.log(`You should eat a ${food}`));
```

Comment:
Hm, I think we should come up with an example that does some side effect in the `forEach` block, like setting a property in the items of the array.
What if each fruit was an object, and we had an `eaten` property that we toggle to true?